apps/judicial-system/web/src/utils/hooks/useVerdict/verdict.graphql

```diff
@@ -2,6 +2,7 @@ query Verdict($input: VerdictQueryInput!) {
   verdict(input: $input) {
     id
     created
+    modified
```

Comment:
Is this attribute used in the frontend?

Reply:
No, added it to trigger the codegen in the build script on github 🫠 Will remove in a follow up PR